### PR TITLE
Prettify charts.

### PR DESCRIPTION
**Pull request by fnussber**
A facelift of the ITC charts in the OT and the web app. I also made the charts in the web app a tad bit bigger and did some minor refactoring. No actual changes in functionality, this are purely visual improvements.

I recycled the color schema and general look of the QV charts, I think it makes sense if charts throughout the OT use the same "visual language".

image

image

### CODE REVIEW

File: bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/PrinterBase.java [hunk 1]
```diff
@@ -77,14 +77,13 @@ protected void _printImageLink(final UUID id, final SpcChartType type, final Plo
     }
 
     protected void _printImageLink(final UUID id, final SpcChartType type, final int index, final PlottingDetails pd) {
-        _print("<IMG alt=\"" + toImgAlt(type, index) + "\" height=500 src=\"" + ServerInfo.getServerURL() +
+        _print("<img alt=\"" + toImgAlt(type) + "\" src=\"" + ServerInfo.getServerURL() +
```
Comment (cquiroz) [inline bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/PrinterBase.java]: No `width` or `height` attributes on ``? Or does `toPlotLimits` fill those attributes?

**Comment by cquiroz** [comment]
This maybe a stupid question since I don't know anything about the ITC. Why on the first graphs there are lines with value 0, like `Single Exp S/N`?

File: bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/PrinterBase.java (hunk 1 above)
Comment (fnussber) [inline bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/PrinterBase.java]: ```width``` and ```height``` are  not mandatory, so I though as long as I just want to display the chart in its original size without any further layout restrictions I can just as well omit them.

**Comment by fnussber** [comment]
This is the effect of a filter that blocks light above ~560 nm.

File: bundle/edu.gemini.itc.shared/src/main/java/edu/gemini/itc/shared/ITCChart.java [hunk 2]
```diff
@@ -4,29 +4,89 @@
...
 public final class ITCChart {
-    private final XYSeriesCollection seriesData = new XYSeriesCollection();
-    private final JFreeChart chart;
+
+    // ==========================================================================
+    // Static
+
+    // Define a set of colors for charts.
+    // This are the same colors as used for Queue Visualization, maybe at some point we want
+    // to make the QV color schematas available OT wide?
```
Comment (swalker2m) [inline bundle/edu.gemini.itc.shared/src/main/java/edu/gemini/itc/shared/ITCChart.java]: That makes sense to me.

**Comment by swalker2m** [comment]
:+1:

**Pull request merged**